doc/Tex/RAM_SCB_testing.tex

```diff
@@ -1,77 +1,57 @@
 \begin{enumerate}
 \item{Compile the code with any options required by the particular test.}
-\item{Create a run directory entitled \verb*lrun_testl.  If on exists, it will 
-  be deleted, so use caution when using this name for any other purpose 
-  besides automated testing.}
-\item{Copy the required input and parameter files into the new test run 
-  directory.}
-\item{Run the code.  Test simulations are only long enough to perform the 
-  features being tested.}
-\item{Perform a specialized version of \verb*ldiffl, included in the 
-  distribution, to compare the results produced by the test simulation against 
-  reference solutions stored in the output directory of the installation 
-  location.  If the two files have values that differ by a certain amount 
-  (default is an absolute difference of $10^{-30}$), the test will fail.}
+\item{Create a run directory entitled \verb*lrun_testl.  If one exists, it will be deleted, so use caution when using this name for any other purpose besides automated testing.}
+\item{Copy the required input and parameter files into the new test run directory.}
+\item{Run the code. Test simulations are only long enough to perform the features being tested.}
+\item{Perform a specialized version of \verb*ldiffl, included in the distribution, to compare the results produced by the test simulation against reference solutions stored in the output directory of the installation location. If the two files have values that differ by a certain amount (default is an absolute difference of $10^{-9}$), the test will fail.}
 \end{enumerate}
 
-Each of these steps can be called individually by using {\tt make}
-\textit{nametest}\_\textit{stepname}, where the step names are {\tt compile},
-{\tt rundir}, {\tt run}, and {\tt check}.
+Each of these steps can be called individually by using {\tt make} \textit{nametest}\_\textit{stepname}, where the step names are {\tt compile}, {\tt rundir}, {\tt run}, and {\tt check}.
 
-If, at any point, the test fails, the details will be recorded in 
-\textit{nametest}.diff in the installation directory.  If the test is 
-successful, this file will be created but be empty.  Tests that fail at the 
-comparitive stage will list all of the differences.  When this happens, 
-additional tools to examine the file differences are recommended.  An 
-excellent visualization of file differences can be rendered with 
-\verb*ltkdiffl, an open source tool that is already installed on many 
-Linux machines.
+If, at any point, the test fails, the details will be recorded in \textit{nametest}.diff in the installation directory. If the test is successful, this file will be created but be empty. Tests that fail at the comparitive stage will list all of the differences. When this happens, additional tools to examine the file differences are recommended. An excellent visualization of file differences can be rendered with \verb*ltkdiffl, an open source tool that is already installed on many Linux machines.
```

Comment:
`vimdiff` or `vim -d` (they're basically equivalent) is a good command line option. Similarly, `gvimdiff` is much lighter weight than tkdiff but is also gui based.